debian/Debianize.hs

```diff
@@ -42,7 +42,7 @@ main = performDebianization customize
 
              -- Add dependencies to the binary debs.
              (debInfo . binaryDebDescription (BinPkgName "cabal-debian") . relations . depends) %=
-                (++ (rels "apt-file, debian-policy, debhelper, haskell-devscripts (>= 0.8.19)"))
+                (++ (rels "apt-file, debian-policy, dh-sequence-haskell, haskell-devscripts (>= 0.16.8)"))
```

Comment:
What are the impediments to not doing as the commit message says?

Reply:
Probably none—except I am not sure how to pull the desired `compat` value from [Policy.hs](https://github.com/clinty/cabal-debian/blob/master/src/Debian/Policy.hs#L122).

Reply:
Hi, does this file (in `./debian`) describe `cabal-debian` or the target being debianized?